deploy.sh

```diff
@@ -84,6 +84,8 @@ build_push_images() {
 
 if [ "${TRAVIS_BRANCH}" == 'master' ]; then
 	export ENV_CSI_IMAGE_VERSION='canary'
+if [ "${TRAVIS_BRANCH}" == 'release-v3.0' ]; then
```

Comment:
target branch need to be changed to `release-v3.1`

Reply:
done.